introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I'm Louis Fyne. I'm 6'3", and maintain a very consistent panda bear shape.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.